generate_data/jzarr/generate_data.sh

```diff
@@ -0,0 +1,17 @@
+# cd to this directory
+# https://stackoverflow.com/a/6393573/2700168
+cd "${0%/*}"
+
+set -e
+set -u
+set -x
+
+mvn clean package
```

Comment:
```suggestion
mvn --no-transfer-progress clean package
```
I think adding this option will suppress the thousands of lines related to file transfer progress in the CI logs, making them more readable.

Reply:
Alternatively, or in addition, we might consider caching the `~/.m2/repository` folder on GitHub Actions (e.g. https://stackoverflow.com/a/59571393)

Reply:
I'll add those in here, @grlee77, and then hand this off to you for a final blessing.